app/scripts/modules/core/src/widgets/spelText/SpelText.tsx

```diff
@@ -33,7 +33,7 @@ export class SpelText extends React.Component<ISpelTextProps, ISpelTextState> {
   private autocompleteService: SpelAutocompleteService;
   private readonly spelInputRef: any;
   private destroy$ = new Subject();
-  private $input: JQLite;
```

Comment:
This change was needed unfortunately. `JQLite` from angular is only defined with `@types/angular@1.6.43` and due to some yarn magic it ended up being made available to `core`. With the migration to workspace, that version can no longer be used since the majority of `core` relies on `@types/angular@1.6.26` and I had to explicitly use that version using `resolutions`.